onnx/checker.cc

```diff
@@ -127,6 +140,31 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
     for (const StringStringEntryProto& entry : tensor.external_data()) {
       if (entry.has_key() && entry.has_value() && entry.key() == "location") {
         has_location = true;
+#ifdef _WIN32
+        const fs::path windows_path(entry.value());
```

Comment:
This would not work. 
Please manually use https://docs.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-multibytetowidechar function to convert entry.value() to a wide char string, and during the conversion you need to set code page to CP_UTF8. Because the ONNX standard expects all strings are UTF-8.

Reply:
You could try if it works for paths with Chinese characters.

Reply:
Good catch. I should further use `utf8str_to_wstring` (including `MultiByteToWideChar`) to convert the std::string into std::wstring first.